src/scrapers/union-bank.ts

```diff
@@ -210,7 +210,7 @@ async function searchByDates(page: Page, startDate: Moment) {
 
 async function getAccountNumber(page: Page) {
   const selectedSnifAccount = await page.$eval('#ddlAccounts_m_ddl option[selected="selected"]', (option) => {
-    return (option as HTMLElement).innerText;
+    return (option as HTMLElement).innerText.replace(/[^ -~]+/g, '');
```

Comment:
@orentet why is this needed?
missing an explanation for this change in your PR

Reply:
when selecting the account, it contains non-printable chars on both sides, which this code deletes.

Reply:
maybe I'm wrong but when using `[^ -~]+` pattern (starting with ^ symbol in the brackets) you are basically replacing everything that is not one of those characters, so you keep those signs and not the actual account value. where am I mistaken?

regarding that non-printable sign, I tried your regexp in two online editors and that sign broke them. can you instead use the hex representation of that sign, or maybe use allowed list instead. for example `/[\d-]+/g`